apps/vaults/contexts/useStakingRewards.tsx

```diff
@@ -130,7 +130,7 @@ export const StakingRewardsContextApp = memo(function StakingRewardsContextApp({
 		const calls = [];
 		for (const {address} of stakingRewards) {
 			const baseContract = {
-				address: toWagmiAddress(address),
+				address: address,
```

Comment:
Whenever we have the same name I'd go for this for making it easier to read

```suggestion
				address,
```